draft-ietf-shmoo-remote-fee.md

```diff
@@ -97,7 +97,14 @@ probably written with mainly having email interactions in mind when talking abou
 This document extends this principle to explicitly cover online
 participation at meetings.
 
-In order to fully remove barriers to participation, any free
+However, this document does not automatically advise that all kind of IETF meetings and events
```

Comment:
Maybe "This document does not recommend that all types of IETF meetings and events must have a remote participation option, as there could be ..."

Reply:
Actually I didn't want to use "recommend" because enabling remote participation whenever possible seems like a good recommendation. I was initially using "require" instead of "advise" but that seemed too strong...

Reply:
Okay, I appreciate your concerns about advise/require/recommend.  I just think changing the sentence structure to "This doc doesn't ... because ..." is better.

Reply:
Not sure which change you are looking for now. Can you make another proposal?

Reply:
"This document does not advise that all IETF meetings or events must have a remote participation item, because there could be technical or other reasons why that might not always be possible."